examples/oidc/james/jmap.properties

```diff
@@ -17,7 +17,7 @@ tls.secret=james72laBalle
 # The following entry specify the link to the URL of the public key file,
 # which should be a PEM format file.
 #
-#jwt.publickeypem.url=file://conf/jwt_publickey
+jwt.publickeypem.url=file://conf/jwt_publickey
```

Comment:
Why?

Reply:
I run the docker-compose oidc example on master and James exited with an error when reading jmap.properties "publickey" is mandatory. I see jmap draft authentication strategies need a public key anyway so I add a public key here.

Reply:
Please follow the instructions there to build the memory image: https://github.com/apache/james-project/tree/master/server/apps/memory-app#docker-distribution

Cc @Arsnael

Reply:
I'm surprised you had issues... normally the latest memory image should be good since my work has been merged